src/wallet/db/level/db.ts

```diff
@@ -39,24 +38,20 @@ export class WalletDB implements DbInterface {
         await this.db.put(wdb.V, version.toString());
     }
 
-    public async getMasterKey(): Promise<{
-        privateKey: Buffer;
-        chaincode: Buffer;
-    }> {
+    public async getMasterKey() {
```

Comment:
Removed return type, as its redundant since already declared in `src/wallet/db/db.interface.ts`